src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -286,4 +275,83 @@ void getHearingChannels_ifAdjourmentEnabled_getNextHearing() {
             .containsOnly(TELEPHONE);
     }
 
+    @DisplayName("When adjournment flag is false, returns hearing chanel from the case")
```

Comment:
Couple of typos in the display names: chanel -> channel

Reply:
Updated the type error.

Reply:
Removed the unwanted test case